core/workflow-core/src/main/scala/edu/uci/ics/amber/core/storage/result/ResultStorage.scala

```diff
@@ -0,0 +1,15 @@
+package edu.uci.ics.amber.core.storage.result
+
+import edu.uci.ics.amber.virtualidentity.WorkflowIdentity
+
+object ResultStorage {
```

Comment:
add some comments about the purpose of this Singleton and its limitation.

Reply:
done